talent/application-process.md

```diff
@@ -0,0 +1,69 @@
+# Applying at source{d} 
+
+This document describes how the developers are able to apply at source{d} and the hiring process. 
+
+
+### Visiting our careers page
+
+- All the open roles can be found at https://sourced.tech/careers/ where the developer is able to apply to any desired role. 
+
+- If there is not a fitting role, but the developer is interested in working with us, they could send an email to talent@sourced.tech with their GitHub account and a small intro about themselves. 
+
+### Screening  
+
+- In less than a week after the submission of the application the developer receives an email with feedback, however there can be delays. In any case, the developer is free to reach out to us at any moment at talent@sourced.tech.
+
+- In case of positive feedback in the application, the developer is asked to move to a quick Skype/Zoom/Phone call with the lead of the applied team. 
+
+### Skype/Zoom/Phone call  
+
+- The developer is having a quick call with the lead engineer of the team he applied to. This should be a general discussion that can last between 15-30 minutes. It's purpose is to know the background and experience of the developer, as well as the developer to know source{d} better. It's expected that the developer has already done a basic research about source{d} and has specific questions about the role and/or the company to be expressed during the call. 
+
+- There are noteable exceptions to this, such as higher volume junior roles where we directly ask the developer to do the coding challenge before a call.
+
+### Coding Challenge
+
+- Every developer who is passing successfuly the first interview, should complete the coding challege, remotely.
+
+- The coding challenge can be solved in Go/Python/JavaScript/Scala/Ruby while the developer has already been asked.
```

Comment:
This phrase doesn't make much sense. While the developer has already been asked what?